examples/main/main.cpp

```diff
@@ -10,6 +10,9 @@
 #include <cstring>
 #include <ctime>
 #include <fstream>
+#if __cplusplus >= 201703L
+#include <filesystem>
+#endif
```

Comment:
Considering that we build for C++11, is this ever going to be true?

Reply:
not anytime soon. I hope we will not target C++11 forever though? c++17 is already 7 years old

Reply:
Maybe so, but that's a different discussion. I don't see the point of including code that is never going to be used.